static/src/stylesheets/module/_social.scss

```diff
@@ -176,7 +178,12 @@ category: Common
     background-color: colour(social-pinterest);
 
     // The pinterest icon needs to be centered within the span due to its small width/height
-
+    svg {
```

Comment:
This Pinterest icon has always caused problems as it's not aligned the same as the other share icons. Maybe @Pearson82 could provide us one with the same spacing so we can remove all this CSS?